es6.md

```diff
@@ -145,3 +145,35 @@ interface ClassExpression <: Class, Expression {
     type: "ClassExpression";
 }
 ```
+
+## ExportDeclaration
+
+```js
+interface ExportDeclaration <: Node {
+    default: boolean;
+    declaration: VariableDeclaration | Declaration | Expression | null;
```

Comment:
Can remove `VariableDeclaration` as that's a declaration :-). Would also need #39 to land for this to be accurate, so that should be blocking this.